Add activestate OIDC docs (#15548)
pypi/warehouse: docs/user/trusted-publishers/adding-a-publisher.md

=== "ActiveState"

>     For ActiveState, you need to provide the project name on ActiveState's Platform,

Reviewer: ```suggestion
    For ActiveState, you need to provide the name of the ActiveState project,
```
I think this makes it a bit more clear which project name this is.

Author: I agreed!

>     the organization that project belongs to, and the username performing

Reviewer: ```suggestion
    the ActiveState organization that project belongs to, and the ActiveState user performing
```

>     the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).

Reviewer: I'm assuming this URL will start resolving sometime after we enable this feature?

Reviewer 2: Yes, correct. It should be live soon. We'll get it set up so that it can be viewable before we go live.
